master/internal/rm/multirm/multirm_intg_test.go

```diff
@@ -733,7 +734,15 @@ func TestGetRMName(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			rmName, err := mockMultiRM.getRM(tt.rmName, tt.rpName)
 			require.Equal(t, tt.expectedRMName, rmName)
-			require.Equal(t, tt.err, err)
+			if tt.err != nil && strings.Contains(tt.err.Error(), "exists for both resource managers") {
+				// Manually checking for ErrRMConflict because RMs may be returned in different order.
+				require.ErrorContains(t, err, "exists for both resource managers")
+				require.ErrorContains(t, err, "aws")
+				require.ErrorContains(t, err, "gcp")
```

Comment:
weird to hard code it this way to me. should it just use parameters in the test case instead?

Reply:
I could introduce an extra parameter to use just in this case.
is that preferable? even if the other 7/8 cases won't need it ?